packages/components/drop-down-item/styled-drop-down-item.js

```diff
@@ -50,8 +50,13 @@ const StyledDropdownItem = styled.div`
   padding: ${(props) =>
     props.isModern ? "0 8px" : props.theme.dropDownItem.padding};
   line-height: ${(props) => props.theme.dropDownItem.lineHeight};
-  box-sizing: border-box;
   text-align: left;
+  ${(props) =>
+    props.theme.interfaceDirection === "rtl" &&
+    css`
+      text-align: right;
```

Comment:
Why override "text-align" when you can put a condition on the 53 line

Reply:
yes it is not necessary deleted